src/frontend/packages/cf-autoscaler/src/cf-autoscaler.module.ts

```diff
@@ -0,0 +1,43 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { NgxChartsModule } from '@swimlane/ngx-charts';
+import { of } from 'rxjs';
+
+import { CoreModule } from '../../core/src/core/core.module';
+import { MDAppModule } from '../../core/src/core/md.module';
+import { SharedModule } from '../../core/src/shared/shared.module';
+import { AutoscalerModule } from './core/autoscaler.module';
+import { AutoscalerTabExtensionComponent } from './features/autoscaler-tab-extension/autoscaler-tab-extension.component';
+
+const customRoutes: Routes = [
+  {
+    path: 'autoscaler',
+    loadChildren: './core/autoscaler.module#AutoscalerModule',
+    data: {
+      stratosNavigation: {
+        text: 'Applications',
```

Comment:
Do you mean to move all of the autoscaler path into the `applications` path?
Change the paths of:
```
https://localhost:4200/autoscaler/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/edit-autoscaler-policy
https://localhost:4200/autoscaler/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/app-autoscaler-metric-page
https://localhost:4200/autoscaler/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/app-autoscaler-scale-history-page
```
To:
```
https://localhost:4200/applications/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/edit-autoscaler-policy
https://localhost:4200/applications/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/app-autoscaler-metric-page
https://localhost:4200/applications/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/app-autoscaler-scale-history-page
```

Reply:
Do you mean to remove the below code?
```
    data: {
      stratosNavigation: {
        text: 'Applications',
        matIcon: 'apps',
        position: 20,
        hidden: of(true)
      }
    },
```
But when i remove this part of code, the `https://localhost:4200/autoscaler/b5F2dF-Qr-r7yNWeDya0GTOL09c/2369e308-0733-40ce-ae9e-ed85fdbcca40/edit-autoscaler-policy` and other autoscaler paths will failed to load with 404 error.
I don't know why... Could you please help me check? @richard-cox @KlapTrap

Reply:
Yeah, that's what I thought. I think it would be worth rich and I taking a look and seeing what's going on, it doesn't seem right for you to have to do that!